src/demo/utils/image.ts

```diff
@@ -0,0 +1,13 @@
+import { Image } from 'image-js';
+
+export function imageToMask(image: Image) {
+  switch (image.colorModel) {
+    case 'GREY':
+    case 'BINARY':
+      return image.threshold();
```

Comment:
If it's binary, shouldn't we directly return it?

Reply:
I wasn't sure about this case. If I return it directly it's still an `Image` and not a `Mask` right?

Reply:
AFAIK an `Image` with a `BINARY` colorModel isn't possible/doesn't make sense.